### DISCUSSION TITLE
Unable to access my GitHub account

### DISCUSSION THREAD

**Discussion by anshulsharma1710** [community/community#137611] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Few days back suddenly I've got an mail from GitHub that my account got some suspicious activity so they forcely suggest me to change the password. But after changing the password they've suspended my account and I'm unable to access my account.
I'm writing this mail from my secondary account due to access denied issue.
I've raised ticket on GitHub support also but didn't got any reply from them.
Please anybody faces same issue and got the solution please help me to get my account back.

**Comment by ebndev** [comment]
Hi @anshulsharma1710,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
